sparklingpandas/pcontext.py

```diff
@@ -175,47 +220,80 @@ def frame_to_rows(frame):
         return sp_df
 
     def sql(self, query):
-        """Perform a SQL query and create a L{Dataframe} of the result."""
+        """Perform a SQL query and create a L{Dataframe} of the result.
+        The SQL query is run using the local Spark SQL context. This
+        is not intended for querying arbitrary databases, but rather querying
+        Spark SQL tables.
+        Parameters
+        ----------
+        query: string
+            The SQL query to pass to Spark SQL to execute.
+        Returns
+        -------
+        Sparkling Panda's Dataframe.
+        """
         return Dataframe.from_spark_rdd(self.sql_ctx.sql(query), self.sql_ctx)
 
     def table(self, table):
-        """Returns the provided table as a L{Dataframe}"""
+        """Returns the provided Spark SQL table as a L{Dataframe}
+        Parameters
+        ----------
+        table: string
+            The name of the Spark SQL table to turn into a L{Dataframe}
+        Returns
+        -------
+        Sparkling Panda's Dataframe.
+        """
         return Dataframe.from_spark_rdd(self.sql_ctx.table(table),
                                         self.sql_ctx)
 
-    def from_spark_rdd(self, spark_rdd, sql_ctx):
+    def from_spark_rdd(self, spark_rdd):
         """
-        Translates a Spark DataFrame Rdd into a SparklingPandas dataframe.
-        :param dataframe_rdd: Input dataframe RDD to convert
-        :return: Matchign SparklingPandas dataframe
+        Translates a Spark Dataframe into a SparklingPandas Dataframe.
+        Currently, no checking or validation occurs.
+        Parameters
+        ----------
+        spark_rdd: Spark Dataframe
+            Input Spark Dataframe.
+        Returns
+        -------
+        Sparkling Panda's Dataframe.
         """
-        return Dataframe.from_spark_rdd(spark_rdd, sql_ctx)
+        return Dataframe.from_spark_rdd(spark_rdd, self.sql_ctx)
 
     def DataFrame(self, elements, *args, **kwargs):
```

Comment:
I meant  in our internal docs. But, yes, I think we should change this capitalization inconsistency between our lib and spark, pandas. I agree it should probably be in a separate change.